README.md

```diff
@@ -1,65 +1,277 @@
+
+### Generalised Cost Matrix
+This cost matrix contains the mean (or median) generalised cost calculated from each of the
+itineraries provided by the OTP server. This output is saved as a CSV in the square format at
+whatever zone system was provided in the input config file, the first row and column contain the
+zone IDs from the zone centroids input.
+
+### Cost Metrics
+A second CSV output is the cost metrics CSV, this contains a variety of cost 
+
+The first six columns contain the name, ID and zone system for the origin and destination zones.
+The seventh column contains the total number of itineraries OTP found between the two zones. The
+remaining columns are various statistics about the following metrics:
+- duration: total duration of the route, in seconds.
+- walkTime: total time spent walking on the route, in seconds.
+- transitTime: total time spent in transit, in seconds.
+- waitingTime: total waiting time, in seconds.
+- walkDistance: total walk distance on the route, in metres.
+- otp_generalised_cost: the generalised cost calculated by OTP to determine how to rank the
+  itineraries found.
+- transfers: total number of transfers in the route.
+- generalised_cost: generalised cost calculated for each itinerary using the parameters from the
+  input config file.
+- startTime: the departure time of the route.
+- endTime: the arrival time of the route.
+
+### Response Data
+This file contains all the data provide in OTP's routing response, it is saved as a JSON lines
+file i.e. each line is a single response for a pair of zones in JSON format. This file contains
+all the individual itineraries and their respective statistics that OTP calculated.
+
+The file can be opened in any basic text editor, although it is recommended to use Python if any
+further analysis on the file is required. The `CostResults` class provided in the `otp4gb\cost.py`
+module can be used to parse the individual lines from the file, to perform any bespoke analysis.
+Additional a `cost_matrix_from_responses` function is provided (in the same cost module) to 
+use the responses file to recalculate the generslised cost matrix with different parameters and
+recreate the CSV outputs.
+
+## Post Processing
+This sections outlines some stand-alone post-processing scripts to perform additional analysis
+on the outputs from OTP4GB-Py.
+
+### Accessibility
+The [accessibility](accessibility.py) script uses the zone to zone average durations to calculate
+the amount of various land use metrics which can be reached from a zone in a given time limit. The
+inputs for this script are provided in a config file and outlined in table 3, (see 
+[accessibility config](config/accessibility_config.yml) for an example).
+
+**Table 3:** *Details of the config file parameters for the accessibility script.*
+| Parameter                  | Type               | Default | Description                                                                                                      |
```

Comment:
Newline required before table

Reply:
Fixed in 73014cf3fff0ce7ecfc1b0144123aafe469d019c